rust/zypp-agama/zypp-agama-sys/c-layer/Makefile

```
@@ -10,9 +10,15 @@ clean:
 	rm -vf *.o *.a
 
 check:
+	git ls-files | grep '\.[ch]' | \
```

```suggestion
	git ls-files | grep -E '\.(c|h|cxx|hxx)$' | \
```

need better matching or it matches unrelated javascript files from the frontend